src/Runner.jl

```diff
@@ -69,31 +74,6 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
     target = aatriplet(platform)
     host_target = aatriplet(host_platform)
 
-
-    # If the ABI-agnostic triplets of the target and the host platform are the
-    # same, then we have to be very careful: we can't have distinct wrappers, so
-    # we have to be sure that their ABIs are consistent and that we're correctly
-    # writing the wrappers for the target platform.  In  particular, what we care
-    # about with regard to the wrappers is the C++ string ABI:
-    #   * they are equal: this is fine
-    #   * they're different and the host has a preference for the C++ string ABI:
-    #     we can't deal with this situation as the host wrappers will be always
-    #     overwritten, then error out
-    #   * in all other cases we don't do anything here, below we'll let
-    #     the host wrappers be overwritten by the wrappers for the target
-    if target == host_target
-        target_cxxabi = cxxstring_abi(platform)
-        host_cxxabi   = cxxstring_abi(host_platform)
-        if target_cxxabi !== host_cxxabi
-            if host_cxxabi !== nothing
-                # This is a very unlikely situation as ideally the host
-                # shouldn't have particular preferences for the ABI, thus in
-                # practice we should never reach this.
-                error("Incompatible C++ string ABIs between the host and target platforms")
-            end
-        end
-    end
```

Comment:
We're now correctly separating the wrappers between the host and the target, so we don't really need this guard anymore